# id;timestamp;commentText;codeText;commentWords;codeWords
MultiSearchResponseTests -> public void testFromXContentWithFailures() throws IOException;1531179852;Test parsing {@link MultiSearchResponse} with inner failures as they don't support asserting on xcontent equivalence, given that_exceptions are not parsed back as the same original class. We run the usual {@link AbstractXContentTestCase#testFromXContent()}_without failures, and this other test with failures where we disable asserting on xcontent equivalence at the end.;public void testFromXContentWithFailures() throws IOException {_        Supplier<MultiSearchResponse> instanceSupplier = MultiSearchResponseTests::createTestInstanceWithFailures__        _        _        boolean supportsUnknownFields = true__        _        boolean assertToXContentEquivalence = false__        AbstractXContentTestCase.testFromXContent(NUMBER_OF_TEST_RUNS, instanceSupplier, supportsUnknownFields, Strings.EMPTY_ARRAY,_                getRandomFieldsExcludeFilterWhenResultHasErrors(), this::createParser, this::doParseInstance,_                this::assertEqualInstances, assertToXContentEquivalence, ToXContent.EMPTY_PARAMS)__    };test,parsing,link,multi,search,response,with,inner,failures,as,they,don,t,support,asserting,on,xcontent,equivalence,given,that,exceptions,are,not,parsed,back,as,the,same,original,class,we,run,the,usual,link,abstract,xcontent,test,case,test,from,xcontent,without,failures,and,this,other,test,with,failures,where,we,disable,asserting,on,xcontent,equivalence,at,the,end;public,void,test,from,xcontent,with,failures,throws,ioexception,supplier,multi,search,response,instance,supplier,multi,search,response,tests,create,test,instance,with,failures,boolean,supports,unknown,fields,true,boolean,assert,to,xcontent,equivalence,false,abstract,xcontent,test,case,test,from,xcontent,instance,supplier,supports,unknown,fields,strings,get,random,fields,exclude,filter,when,result,has,errors,this,create,parser,this,do,parse,instance,this,assert,equal,instances,assert,to,xcontent,equivalence,to,xcontent
MultiSearchResponseTests -> public void testFromXContentWithFailures() throws IOException;1548943934;Test parsing {@link MultiSearchResponse} with inner failures as they don't support asserting on xcontent equivalence, given that_exceptions are not parsed back as the same original class. We run the usual {@link AbstractXContentTestCase#testFromXContent()}_without failures, and this other test with failures where we disable asserting on xcontent equivalence at the end.;public void testFromXContentWithFailures() throws IOException {_        Supplier<MultiSearchResponse> instanceSupplier = MultiSearchResponseTests::createTestInstanceWithFailures__        _        _        boolean supportsUnknownFields = true__        _        boolean assertToXContentEquivalence = false__        AbstractXContentTestCase.testFromXContent(NUMBER_OF_TEST_RUNS, instanceSupplier, supportsUnknownFields, Strings.EMPTY_ARRAY,_                getRandomFieldsExcludeFilterWhenResultHasErrors(), this::createParser, this::doParseInstance,_                this::assertEqualInstances, assertToXContentEquivalence, ToXContent.EMPTY_PARAMS)__    };test,parsing,link,multi,search,response,with,inner,failures,as,they,don,t,support,asserting,on,xcontent,equivalence,given,that,exceptions,are,not,parsed,back,as,the,same,original,class,we,run,the,usual,link,abstract,xcontent,test,case,test,from,xcontent,without,failures,and,this,other,test,with,failures,where,we,disable,asserting,on,xcontent,equivalence,at,the,end;public,void,test,from,xcontent,with,failures,throws,ioexception,supplier,multi,search,response,instance,supplier,multi,search,response,tests,create,test,instance,with,failures,boolean,supports,unknown,fields,true,boolean,assert,to,xcontent,equivalence,false,abstract,xcontent,test,case,test,from,xcontent,instance,supplier,supports,unknown,fields,strings,get,random,fields,exclude,filter,when,result,has,errors,this,create,parser,this,do,parse,instance,this,assert,equal,instances,assert,to,xcontent,equivalence,to,xcontent
